var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -55,15 +58,17 @@ class PyH5py(PythonPackage):
     depends_on("py-setuptools", type="build")
 
     # Build and runtime dependencies
-    depends_on("py-numpy@1.17.3:", type=("build", "run"), when="@3.9:")
-    depends_on("py-numpy@1.19.3:", type=("build", "run"), when="@3:3.5 ^python@3.9.0:")
-    depends_on("py-numpy@1.17.5:", type=("build", "run"), when="@3:3.5 ^python@3.8.0:3.8")
-    depends_on("py-numpy@1.14.5:", type=("build", "run"), when="@3:")
-    depends_on("py-numpy@1.7:", type=("build", "run"), when="@:2")
+    depends_on("py-numpy@1.17.3:", type=("build", "run"), when="@3.11:")
+    depends_on("py-numpy@1.17.3:2", type=("build", "run"), when="@3.9:3.11")
```

Comment:
```suggestion
    depends_on("py-numpy@1.17.3:1", type=("build", "run"), when="@3.9:3.11")
```

Double check this, I may have misunderstood how Spack's version ranges work, but looking at other ones in this file, I think they're inclusive, so `:1` allows any 1.x.y.

Reply:
That is correct (see https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges).

Reply:
Fixed. Thanks for catching this!